cmd/cloudAuth/main.go

```diff
@@ -112,16 +118,43 @@ func main() {
 
 	api.DatabaseGetDatabaseHandler = getDatabase(storage)
 
-	handler := cors.New(cors.Options{
-		AllowedMethods: []string{"GET", "POST", "PUT", "DELETE"},
-		AllowedHeaders: []string{"Authorization", "Content-Type"},
-		Debug:          true,
-	}).Handler(api.Serve(nil))
-
-	server.SetHandler(handler)
-
-	if err := server.Serve(); err != nil {
-		logger.Fatal().Err(err).Msg("Failed to start server")
+	// handler := cors.New(cors.Options{
```

Comment:
We need CORS to have functional frontend.

Reply:
ofc, this comment out is here by mistake